src/components/search/curated/node/index.tsx

```diff
@@ -1,17 +1,101 @@
 import React from 'react'
+import {NextSeo} from 'next-seo'
+import Topic from '../../components/topic'
 import nodePageData from './node-page-data'
```

Comment:
We're still loading the data statically. Sanity data can't be sourced within a component, only inside `/pages`. 

Should we be sourcing data from Sanity for the topic pages? 

What's the balance between making sure that the work gets done and creating a new pattern?